### DISCUSSION TITLE
got an error: in creating graph from extracted entitity(create_base_entity_graph)

### DISCUSSION THREAD

**Discussion by NavaneethaCSR** [microsoft/graphrag#1256] [Q&A]
Logging enabled at C:\Users\LENOVO\GRAPHRAG\ragtest\output\indexing-engine.log
C:\Users\LENOVO\GRAPHRAG\venv\Lib\site-packages\numpy\core\fromnumeric.py:59: FutureWarning: 'DataFrame.swapaxes' is deprecated and will be removed in a future       
version. Please use 'DataFrame.transpose' instead.
  return bound(*args, **kwds)
🚀 create_base_text_units
                                   id                                              chunk                          chunk_id document_ids  n_tokens
0    3bdf8c193f4b72504e0c63696416bb82  Unlike Next, which we'd eaten at the previous ...  3bdf8c193f4b72504e0c63696416bb82          [1]      1066
1    886c5b1b44c4d7352adc614a9ecab771  Probably one of the best meals I've had ever. ...  886c5b1b44c4d7352adc614a9ecab771          [2]        76
2    d5c64a9a59107a194272c3346c74c9bd  Unlike Next, which we'd eaten at the previous ...  d5c64a9a59107a194272c3346c74c9bd          [3]      1066
3    46b47fb2587c0f82757c419510b670ad  Probably one of the best meals I've had ever. ...  46b47fb2587c0f82757c419510b670ad          [4]        76
4    59cb342f30f3ec348ebab4825fab7386  Service was impeccable. Experience and present...  59cb342f30f3ec348ebab4825fab7386          [5]        54
..                                ...                                                ...                               ...          ...       ...
224  533225c99a20936c412a80ae44ec721d  Pros: It's Hot Doug's. I think that says it al...  533225c99a20936c412a80ae44ec721d       [2504]       106
225  18c33be7b4b5ba8046fde71ead0d9c89  This place is incredible. Definitely worth the...  18c33be7b4b5ba8046fde71ead0d9c89       [2505]       109
226  a5cb98f4b1452523169ed305431dc1ec  2 Words - Ahhh mazing Although the wait is rid...  a5cb98f4b1452523169ed305431dc1ec       [2506]       168
227  ce7c33a01c30219dab23baf3768a42c2  I had the game of the week, bison and also had...  ce7c33a01c30219dab23baf3768a42c2       [2507]        38
228  2567e753624adb1e855df85066e73c6d  The duck fat fries are EXCELLENT! This place a...  2567e753624adb1e855df85066e73c6d       [2508]        36

[2522 rows x 5 columns]
🚀 create_base_extracted_entities
                                        entity_graph
0  <graphml xmlns="http://graphml.graphdrawing.or...
🚀 create_summarized_entities
                                        entity_graph
0  <graphml xmlns="http://graphml.graphdrawing.or...
❌ create_base_entity_graph
None
⠏ GraphRAG Indexer
├── Loading Input (InputFileType.csv) - 1 files loaded (0 filtered) ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00 0:00:00
├── create_base_text_units
├── create_base_extracted_entities
├── create_summarized_entities
└── create_base_entity_graph
❌ Errors occurred during the pipeline run, see logs for more details.

logs.json:

    "type": "error",
    "data": "Error executing verb \"cluster_graph\" in create_base_entity_graph: Columns must be same length as key",
    "stack": "Traceback (most recent call last):\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\datashaper\\workflow\\workflow.py\", line 410, in _execute_verb\n    result = node.verb.func(**verb_args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\graphrag\\index\\verbs\\graph\\clustering\\cluster_graph.py\", line 106, in cluster_graph\n    output_df[[level_to, to]] = pd.DataFrame(\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\pandas\\core\\frame.py\", line 4299, in __setitem__\n    self._setitem_array(key, value)\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\pandas\\core\\frame.py\", line 4341, in _setitem_array\n    check_key_length(self.columns, key, value)\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\pandas\\core\\indexers\\utils.py\", line 390, in check_key_length\n    raise ValueError(\"Columns must be same length as key\")\nValueError: Columns must be same length as key\n",
    "source": "Columns must be same length as key",
    "details": null
}
{
    "type": "error",
    "data": "Error running pipeline!",
    "stack": "Traceback (most recent call last):\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\graphrag\\index\\run\\run.py\", line 227, in run_pipeline\n    result = await _process_workflow(\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\graphrag\\index\\run\\workflow.py\", line 91, in _process_workflow\n    result = await workflow.run(context, callbacks)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\datashaper\\workflow\\workflow.py\", line 369, in run\n    timing = await self._execute_verb(node, context, callbacks)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\datashaper\\workflow\\workflow.py\", line 410, in _execute_verb\n    result = node.verb.func(**verb_args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\graphrag\\index\\verbs\\graph\\clustering\\cluster_graph.py\", line 106, in cluster_graph\n    output_df[[level_to, to]] = pd.DataFrame(\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\pandas\\core\\frame.py\", line 4299, in __setitem__\n    self._setitem_array(key, value)\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\pandas\\core\\frame.py\", line 4341, in _setitem_array\n    check_key_length(self.columns, key, value)\n  File \"C:\\Users\\LENOVO\\GRAPHRAG\\venv\\Lib\\site-packages\\pandas\\core\\indexers\\utils.py\", line 390, in check_key_length\n    raise ValueError(\"Columns must be same length as key\")\nValueError: Columns must be same length as key\n",
    "source": "Columns must be same length as key",
    "details": null
}

can some one explain the error and resolve it

**Comment by NathanAP** [comment]
I'm also getting this but its not in every single attempt to create the data.

Upvoting for visibility.

**Reply by alexfrocha** [reply]
me too

**Reply by LG-lub** [reply]
我也出现了这个错误，怎么解决呢？
